autoarena/store/database.py

```diff
@@ -4,7 +4,7 @@
 import duckdb
 
 DATABASE_FILE = Path.cwd() / "autoarena.duckdb"
```

Comment:
Should this also be relative to `Path(__file__)`?

Reply:
No — most users will install this as a package, and we want the `data` directory to be in their working directory rather than next to wherever this source code ended up.

This behavior is documented in the [root](https://github.com/kolenaIO/autoarena) and [data](https://github.com/kolenaIO/autoarena/tree/trunk/data) READMEs 🙂